common/src/main/resources/assets/voicechat/lang/en_us.json

```diff
@@ -1,217 +1,212 @@
+  "message.voicechat.invite_player": "Press to invite %s to your group.",
+  "message.voicechat.press_to_reassign_key": "Press to assign a key",
   "message.voicechat.set_up": "Press %s to set up",
   "message.voicechat.onboarding.reset": "Onboarding status has been reset",
   "message.voicechat.onboarding.next": "Next",
   "message.voicechat.onboarding.confirm": "Confirm",
   "message.voicechat.onboarding.back": "Back",
   "message.voicechat.onboarding.cancel": "Cancel",
   "message.voicechat.onboarding.introduction.title": "Set Up %s",
-  "message.voicechat.onboarding.introduction.description": "This setup guide helps you to configure your voice chat.\n\nIf you know what you are doing, you can skip this.",
-  "message.voicechat.onboarding.introduction.skip": "I know what I'm doing - Skip",
+  "message.voicechat.onboarding.introduction.description": "This setup guide will help you configure your voice chat.\n\nIf you know what to do, you can skip it.",
+  "message.voicechat.onboarding.introduction.skip": "I Know What to Do - Skip",
   "message.voicechat.onboarding.skip.title": "Skip Setup",
-  "message.voicechat.onboarding.skip.description": "Do you really want to skip the setup guide?\n\nIf you skip the setup you have to adjust all settings manually!",
+  "message.voicechat.onboarding.skip.description": "Are you sure you want to skip this setup guide?\n\nIf you skip it, you will have to adjust all the settings manually!",
   "message.voicechat.onboarding.microphone": "Select a Microphone",
-  "message.voicechat.onboarding.speaker": "Select a Speaker",
+  "message.voicechat.onboarding.speaker": "Select Speakers",
   "message.voicechat.onboarding.activation.title": "Select an Activation Method",
-  "message.voicechat.onboarding.activation": "Please choose a method to activate your Microphone.",
-  "message.voicechat.onboarding.activation.ptt": "When selecting %s you need to press and hold a button to talk.",
+  "message.voicechat.onboarding.activation": "Please choose the method to activate your microphone.",
+  "message.voicechat.onboarding.activation.ptt": "When selecting %s, you need to press and hold a button to speak.",
   "message.voicechat.onboarding.activation.ptt.name": "Push to Talk",
-  "message.voicechat.onboarding.activation.voice": "%s will automatically activate your microphone once a certain volume threshold is reached.",
+  "message.voicechat.onboarding.activation.voice": "%s automatically activates your microphone when a certain volume threshold is reached.",
   "message.voicechat.onboarding.activation.voice.name": "Voice Activation",
   "message.voicechat.onboarding.ptt.title": "Choose a Push to Talk Keybind",
   "message.voicechat.onboarding.ptt.description": "Please select the key that should be used to activate your microphone.",
-  "message.voicechat.onboarding.ptt.button_description": "Press to reassign",
+  "message.voicechat.onboarding.ptt.button_description": "Press again to reassign",
```

Comment:
Why again?

Reply:
To me, "Press to reassign" suggests that some key combination is already assigned, and you need to override it by clicking the button below and pressing a key of your choice.

Half the time this is a false assumption, unless the user first checked the key binds, made adjustments there, and then pressed V.

An alternative could be something like "Press the button below to assign a key" or simply "Press to assign". Does that work for you?

Reply:
Depending on the context, that may have been the intent of the message. However, I assume that's not what you meant in this case, so with that in mind, I'll change the text to "Press to assign".